packages/liferay-npm-scripts/src/utils/getPaths.js

```diff
@@ -24,7 +24,7 @@ const readIgnoreFile = require('../utils/readIgnoreFile');
 function getPaths(globs, extensions, ignoreFile) {
 	const root = findRoot();
 
-	ignoreFile = path.join(root || '.', ignoreFile);
+	ignoreFile = ignoreFile ? path.join(root || '.', ignoreFile) : '';
```

Comment:
This tweak allows me to not provide an ignore file to `getPaths`. Basically, we'll end up finding *all* the bad configs in the repo, except for those inside `node_modules` (because we always ignore stuff in there).

When `ignoreFile` is empty (`''`), `fs.existsSync()` returns `false`, and we wind up with an empty `ignores` array, which is what we want.

Further down, `filterChangedFiles` will run, which ensures that in CI we don't error out when somebody submits a PR that doesn't even touch one of these bad config files.